%LaTeX template : http://systbio.org/files/SB_LaTeX_Template_txt_extension.txt
%Author instructions: http://www.oxfordjournals.org/our_journals/sysbio/for_authors/ms_preparation.html

\documentclass[12pt,letterpaper]{article}
\usepackage{natbib}

%Packages
\usepackage{pdflscape}
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{float}
\usepackage{latexsym}
\usepackage{url}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\usepackage{dcolumn}
\usepackage{lineno}
\pagenumbering{arabic}


%Pagination style and stuff
\linespread{2}
\raggedright
\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0} 
\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}
\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}
\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}
\renewcommand{\tableofcontents}{}
%\bibpunct{(}{)}{;}{a}{}{,}

%---------------------------------------------
%
%       START
%
%---------------------------------------------

%Submitting to: Evolution (thorough method + applied data)
%               Proc B (nice story + thorough method in the supplementaries?)
%               MEE (new method (time slicing) + increment on former methods (disparity)) 

% NC: Make sure to format for whichever journal we choose...

\begin{document}

%Running head
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: Tempo and mode in mammalian evolution

\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Cretaceous-Palaeogene extinction does not affect mammalian disparity.} 
% NC: still think this might need some work

\bigskip

\noindent {\normalsize \sc Thomas Guillerme$^1$$^,$$^2$$^*$, and Natalie Cooper$^1$$^,$$^2$$^,$$^3$}\\
\noindent {\small \it 
$^1$School of Natural Sciences, Trinity College Dublin, Dublin 2, Ireland.\\
$^2$Trinity Centre for Biodiversity Research, Trinity College Dublin, Dublin 2, Ireland.\\
$^3$Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7 5BD, UK.}\\
\end{center}
\medskip
\noindent{*\bf Corresponding author.} \textit{Zoology Building, Trinity College Dublin, Dublin 2, Ireland; E-mail: guillert@tcd.ie; Fax: +353 1 6778094; Tel: +353 1 896 2571.}\\
\vspace{1in}

%Line numbering
\modulolinenumbers[1]
\linenumbers

%---------------------------------------------
%
%       ABSTRACT
%
%---------------------------------------------

\newpage
\begin{abstract}
% NC: As ever I'm ignoring this until we are finished with the rest!
%Massive global extinctions have a turn-over effect on biodiversity. When some large group of taxa suffers from a high rate of extinction, it is expected that niches becomes available for potentially unrelated clades that can undergo an adaptive radiation to fill these vacant niches.
%Therefore, in a context of current global biotic and abiotic changes, resolving this question is crucial to understand the effect of mass extinction events on biodiversity.
%The causes and effects of such events are well understood for marine organisms with a good fossil record (e.g. Ammonoidea and Foraminifera) but the effects remains unclear on some iconic vertebrate groups.

%Typically, placental mammals (eutherians) are shown by some studies to be undergoing an adaptive radiation after the Cretaceous-Palaeogene mass extinction event (K-Pg) by originating shortly before the K-Pg event and displaying high morphological evolutionary rates leading to high diversification during the Palaeogene. However, some other studies have demonstrated that eutherians originates during the Cretaceous and don't display significantly high diversification after the K-Pg event.

%Here we propose a new approach to test if eutherians undergo an adaptive radiation after the K-Pg event. We use trees containing both living and fossil taxa based on all the available data (Total Evidence) and the state-of-the-art method in dating (tip dating) along side with a better proxy for niche occupancy (morphological diversity as opposed to taxonomic diversity) and finer grain analysis through time (introducing a time slicing method).

%Our results shows that eutherians don't display significantly higher changes in morphological disparity that expected under a Brownian motion after the K-Pg boundary. We therefore propose that eutherian mammals don't undergo an adaptive radiation during the Palaeogene.

%Why is our stuff better?
%1-More accurate timing (TEM+tip dating vs. molecular node date or morphological parsimony)
%2-Better proxy for niche occupancy (morphological disparity + diversity vs. species diversity) (but niches concept is shite anyway)
%3-Better measure for disparity (centroid distance vs. Foote's quartet)
%4-Two evolutionary models instead of one for looking at disparity through time (punctuated+constant vs. punctuate)
%5-Systematic time units for looking at disparity through time (slices vs. intervals)

\end{abstract}

\noindent (Keywords: disparity, diversity, punctuated equilibrium, gradualism, time slicing)\\
% NC: Note that keywords are things that aren't in the title of the paper
\vspace{1.5in}

\newpage 

%---------------------------------------------
%
%       INTRODUCTION
%
%---------------------------------------------

\section{Introduction}
% 1§ mass extinctions = bad. Loss of species (e.g. P/T 95%). But what comes after is more interesting
Throughout history, life on Earth has suffered a series of mass extinction events resulting in drastic declines in global biodiversity \citep[e.g.][]{RaupPT,BentonPT,rennetime2013,Brusatte2015}.
However, the long-term effects of mass extinctions are more varied \citep{Erwin1998344}, and include increases in species richness in some clades \citep{friedmanexplosive2010}, species richness declines in others \citep{Benton85}, changes in morphological diversity \citep{Ciampaglio2001,Ciampaglio2004,kornextinction2013} and shifts in ecological dominance \citep[e.g.][]{Brusatte12092008,toljagictriassic-jurassic2013,bensonfaunal2014}.
These shifts are characterized by the decline of one clade that is replaced by a different unrelated clade with a similar ecological role (e.g. Brachiopoda and Bivalvia at the end Permian extinction \citealt{Sepkiski1981,CLAPHAM01102006} but see \citealt{Payne22052014}). 
Shifts in ecological dominance are of particular interest because they are a fairly common pattern observed in the fossil record (e.g. Foraminifera; \citealt{D'Hondt01011996,Coxall01042006}; Ichtyosauria; \citealt{thorneresetting2011}; Plesiosauria; \citealt{bensonfaunal2014}) and are often linked to major macroevolutionary processes such as adaptive \citep{Losos2010} or competitive radiations \citep{Brusatte12092008}.

% 2§ explaining the rise of the age of the mammals view.
One classical example of a shift in ecological dominance is at the Cretaceous-Palaeogene (K-Pg) mass extinction 66 million years ago \citep{rennetime2013}, where the non-avian dinosaurs went extinct, potentially leading to the ``rise of the age of the mammals" \citep{archibald2011extinction,Lovergrove}. 
This is based on the idea that placental mammals were able to diversify after the extinction of many terrestrial vertebrates at the K-Pg boundary (including the dominant non-avian dinosaur group; \citealt{luo2007,archibald2011extinction,O'Leary08022013,Brusatte2015}).
Some authors suggest this reflects placental mammals filling the ``empty'' niches left after the K-Pg event \citep{archibald2011extinction}, others suggest it reflects a release from predation and/or competition \citep{Lovergrove}.
However, evidence for the diversification of placental mammals after K-Pg is mixed.
Thorough analysis of the fossil record \citep[e.g.][]{goswamia2011,O'Leary08022013} supports the idea that placental mammals diversified after K-Pg as there are no undebated placental mammal fossils before the K-Pg event and many afterwards \citep{archibald2011extinction,goswamia2011,Slater2012MEE,O'Leary08022013,Wilson2013,Brusatte2015}. 
Conversely, evidence from molecular data suggests that the diversification of placental mammals started prior to the K-Pg extinction event without being drastically affected by it \citep{Douady2003285,bininda2007delayed,meredithimpacts2011,Stadler12042011,beckancient2014}. 
Therefore, whether the diversification of placental mammals began before K-Pg, or in response to the extinctions at K-Pg, is a matter of great debate \citep{O'Leary08022013,Springer09082013,O’Leary09082013}. 

There are three main reasons why there is still debate about the timing of the diversification of placental mammals. In this paper we focus on solving these issues as follows: 
% NC: Might want to look at making this sentence less clunky!
  \begin{enumerate}
    \item \textbf{Palaeontological and neontological data show different patterns.}
    As mentioned above, conclusions about when placental mammals diversified tend to be split depending on what kind of data are used: palaeontological data generally suggest that placental mammals diversified post K-Pg \citep[e.g.][]{O'Leary08022013}, whereas neontological data suggest that K-Pg event had little to no effect on mammalian diversification \citep{bininda2007delayed,meredithimpacts2011,Stadler12042011}. 
    Fortunately we can deal with this issue by using all the data available, rather than using just fossils or molecules. 
    Here we use Total Evidence phylogenies containing cladistic data for both living and fossil taxa along with molecular data for living taxa \citep{eernissetaxonomic1993,ronquista2012}, and using the tip-dating method \citep{ronquista2012,Wood01032013} to get accurate estimates of diversification times for both fossil and living species.
    \item \textbf{Diversity can be defined in different ways.}
    Diversity is a difficult concept to define. 
    In many studies it is measured as taxonomic diversity or species richness \citep{Stadler12042011,meredithimpacts2011,O'Leary08022013}, but often the more interesting aspect of diversity is related to the ecological niches the species occupy \citep{Wesley-Hunt2005,Brusatte12092008,toljagictriassic-jurassic2013}, particularly if we want to  make hypotheses about macroevolutionary processes \citep{Pearman2008149,OlsonRadiation,Losos2010,glor2010phylogenetic}.
    Sometimes taxonomic diversity is used as a proxy for other kinds of diversity, however, species richness can be decoupled from morphological diversity \citep{slaterCetacean,ruta2013,hopkinsdecoupling2013}, so taxonomic diversity may not be the best proxy for ecological diversity.
    In this study we therefore use morphological diversity, also known as disparity \citep[e.g.][]{Wills1994,Hughes20082013},as a way to quantify changes in mammalian diversity that should relate to the ecology of the species.
    \item \textbf{Methods are outdated and make inappropriate assumptions.}
    Many of the methods used to quantify changes in mammalian diversity before and after K-Pg 
    % NC: Actually are these methods used to look at the KPg question? Aren't they general disparity? And oleary etc don't use that at all do they? Rephrase perhaps?
    were proposed \textgreater 20 years ago \citep{Foote01071994,Wills1994} and are sometimes used without modifications \citep[e.g.,][]{brusatte50,Brusatte12092008,cisneros2010,thorneresetting2011,prentice2011,brusattedinosaur2012,toljagictriassic-jurassic2013,ruta2013,bentonmodels2014,bensonfaunal2014}, even when the statistical assumptions of the methods are violated (see Methods).
    Additionally, previous methods are based on an underlying assumption that changes in disparity occur by punctuated evolution \citep[e.g.][]{Wesley-Hunt2005} which is not always the case \citep{Hunt21042015}.
    Finally, most studies of disparity through time use unequal time units based on biostratigraphy \citep{Brusatte12092008,brusattedinosaur2012,toljagictriassic-jurassic2013}. 
    This can be tautological as biostratigraphy is already based on changes in fossil assemblages and morphology through time.
    Here, we modify statistical methods for measuring disparity.
    We also use a time-slicing method that allows us to study disparity through time in a continuous way or by specifying an evolutionary model (punctuated equilibrium or gradual evolution).
  \end{enumerate}

Here, we propose an updated approach to test whether mammals diversified before or after K-Pg, using morphological disparity, measured as cladistic disparity (see Methods), as our proxy for diversity.
We measured the disparity of living and fossil mammals taken from two previously published studies \citep{Slater2012MEE,beckancient2014}. % NC: The details about genes here were what caused my confusion.
Using a novel time-slicing approach we produce fine-grain estimates of disparity through time under two different models of morphological character evolution (either gradual or punctuated). 
Finally, to test whether mammals display significant changes in disparity after the K-Pg boundary, we compared the observed changes to two null models assuming purely stochastic or purely Brownian evolution. 
We found no significant increases in mammalian disparity after the K-Pg event; instead the disparity of placental mammals increased during the K-Pg event. 
These results suggest that the shift in dominant terrestrial vertebrate clades in the fossil record (from non-avian dinosaurs to placental mammals) during the Tertiary was not a direct result of the K-Pg mass extinction.

%---------------------------------------------
%
%       METHODS
%
%---------------------------------------------

\section{Methods}

%To put somewhere:
%Methodological improvements
%Distance matrix: we use Graeme's MORD that more accurately reflects distances between taxa - Actually just Gower from now since Graeme's paper's not out for a while
%Disparity: we use the centroid distance that is a clear and easily-interpretable method that is less dependent from diversity (see supplementaries)
%Time: we use Total Evidence Tip dated trees that are more accurate because they use probabilistic methods to estimate morphological phylogenetic distance (Wright) and provide more accurate ages of diversification events (Ronquist).
%Disparity through time: Finally, we use a *novel method* (to our knowledge) to look at the diversity through time. Instead of calculating the disparity of all species present in a time interval (e.g. Butler and Brusatte), we calculate disparity at a set of arbitrary and equidistant points in time. This new method provides a finer grain resolution of the evolution of diversity through time as well as two well defined models of character evolution (pucntuated - random, acctran, deltran - and continuous - proximity) contrary to the time interval method that assumes only punctuated as a evolutionary model for morphological characters and can be biased towards biostratigraphy.


% NC: I'm not sure that this is necessary for this paper. It's much more linear that the previous one. You could save for the supp mat or your thesis 
%To test whether mammals display a significant change in disparity after the K-Pg boundary we use the following protocol (note that each step is explained in detail below this section; Fig.~\ref{fig_method}.
%\begin{enumerate}
%\item{Data: we used the cladistic morphological matrices and the Total Evidence tip-dated trees published in \citep{Slater2012MEE} and \citep{beckancient2014}.} \\
%\item{Estimating ancestral character states: we used the \citep{Yang01121995} re-rooting method to estimate the states of each characters in the cladistic morphological matrices at each node and created the reconstructed morphological matrix containing observed morphological characters data for tips in the tree and estimated morphological characters states for nodes.}\\
%\item{Calculating the cladisto-space: using the reconstructed morphological matrix we defined the cladisto-space by using a principal components ordination of the Gower distance matrix \citep{Gower71} representing the $n$ finite dimensional space that encompasses the total cladistic morphological variation of the taxa present in the analysis.} \\
%\item{Time slicing: we then separated the cladisto-space into subdivisions containing only the edges (nodes or/and tips) present every 5 million years from present (hereafter called "time-slices").} \\
%\item{Disparity through time: we then calculate the phylogenetic diversity as the number of edges present at each time-slice as well as the cladistic morphological disparity defined as the spread of the edges in the cladisto-space at each time-slice (distance from cladisto-space centroid \citep{finlay2015morphological}.} \\
%\item{Null model testing: finally, we compared the observed disparity to the expected disparity under two different null models: (1) the first one being a completely stochastic model where disparity is random at each point in time.; and (2) the second one being a $Mk_n$ model (as a proxy for Brownian evolution) where disparity increase is constant through time.} \\
%\end{enumerate}

%\begin{figure}[!htbp]
%\centering
%    \includegraphics[keepaspectratio=true]{Figures/MEthod_outline.pdf}
%\caption{General method outline. The grey section represents the Total Evidence data and tip dated trees from two published studies \citep{Slater2012MEE,beckancient2014}. \textbf{A}: We then estimated the ancestral characters states from the observed morphological matrices from both studies. we then calculated the distance matrix using the Gower distance \citep{Gower71} and performed a ordination of the distance matrix to create the cladisto-space. Finally, we divided the cladisto-space matrix using the Time slicing method to measure the changes in morphological disparity through time. \textbf{B}: we then generated two sets of a hundred simulated "null" matrices under two null models, the random matrix: a fully stochastic matrix; and the $Mk_n$ matrix: a matrix simulated under the $Mk_n$ model. We then applied the same procedure as for \textit{A} (ordinated distance matrix and time slicing) to calculate the disparity through time expected for purely random data and for a constant evolution model. Finally we compared our simulated disparity through time to our observed disparity through time to estimate if mammals displayed a significant increase in disparity after the K-Pg boundary.}
%\label{fig_method}
%\end{figure}


\subsection{Cladistic data and phylogenies}
We used the cladistic morphological matrices and the Total Evidence tip-dated trees \citep{ronquista2012} from \citet[][103 taxa and 446 morphological characters]{Slater2012MEE} and \citet[][102 taxa and 421 morphological characters]{beckancient2014}.
We chose these two data sets because they have a similar number of taxa and morphological characters.
\cite{Slater2012MEE} ranges from 310 million years ago (Mya; Late Carboniferous) to the present and focuses on Mammaliamorpha at the family-level.
\cite{beckancient2014} ranges from 170 Mya (Middle Jurassic) to the present and focuses on Eutheria at the genus-level.
We used the first and last occurrences reported in \cite{Slater2012MEE} and \cite{beckancient2014} as the temporal range of each taxon in our analysis.

\subsection{Estimating ancestral character states}
For both datasets we used the re-rooting method \citep{Yang01121995} % NC: Cite Garland and Ives 2000 and maybe felsenstein 2004.
to get Maximum Likelihood estimates of the ancestral states for each character at every node in the tree, using the \texttt{rerootingMethod} function from the R package \texttt{phytools} version 0.4-45 \citep{phytools,R}.
Where there was missing character data for a taxon we followed the method of \cite{Claddis} and treated missing data as any possible observed state for each character.
For example, if a character had two observed states (0 and 1) across all taxa, we attributed the multi-state ``0\&1" value to the taxon with missing data, representing an equal probability of being either 0 or 1.
This allows the ancestral node of a taxon with missing data to be estimated with no assumptions other than that the taxon has one of the observed character states.
%We tested the effect of this assumption against another model where each missing character is treated as ``?'' character. In this second model, the missing data (NA) is treated as a new character ``?'' and the state of the character at the node can then be 0, 1 or ``?''. This method is more conservative because it does not make the assumption that the unobserved characters for taxa with missing data is obligatory one of the states observed for that character. However, simulations show that the first method (treating missing data as multi-state characters) leads to less false positive ancestral state reconstruction than the second (treating missing data as ``?'' character). %link to supp % TG: I'm not sure if that's important here. Nobody cares about this I assume and it has not much to do here. Maybe in the thesis?
% NC: Yeah I'd cut this for now, doesn't even need to go in supp mat. Maybe make a supplementary file just for "in case a reviewer asks for it" stuff. 
%We preferred Maximum Likelihood methods to Bayesian methods because both methods have been shown to produce similar results \citep{royer-carenzichoosing2013} but Maximum Likelihood methods are several orders of magnitude faster than Bayesian ones (total analysis time $\sim$@1.2 CPU year). 
% NC: Again I'd leave this out for now, not really relevant and draws attention to the fact you're not using Bayesian
To % in order to == to !!!
prevent poor % NC: or imprecise? Make sure I've made the correct justification here.
ancestral state reconstructions from biasing our results, especially when a lot of error is associated with the reconstruction, we only included ancestral state reconstructions with a scaled Likelihood $\geq$ than 0.95.
Ancestral state reconstructions with scaled Likelihoods below this threshold were replaced by ``NA''.
%However, for a matter of comparison, we also provided the results of the analysis without removing the ancestral state reconstruction with low scaled likelihoods intervals ($\leq$ 0.95). % link to supp % TG: I think that's unecessary: 'we reran the analysis with crappier values and got crappier results - no shit!'
% NC: Yeah again I'd leave this out as you know it makes no difference.

\subsection{Building the cladisto-space} % NC: Not sure if it's estimating, building, constructing etc.! 
To explore variations in mammalian disparity through time (defined here as the variation in morphologies through time), we use a cladisto-space approach \citep[e.g.][]{Foote01071994,Foote29111996,Wesley-Hunt2005,Brusatte12092008,Hughes20082013,friedmanexplosive2010,toljagictriassic-jurassic2013}. % TG: or just call it morphospace? I think morphospace is less correct as it is not the morpho/metric/logic-space but an approaximation of it through cladistic space but Hughes et al 2008 and others still call it morphospace...
% NC: I think this is a good chance to make this correct! It isn't a morphospace technically.
This approach is similar to constructing a morphospace based on continuous morphological data \citep[e.g.][]{friedmanexplosive2010,finlay2015morphological}, except a cladisto-space is an approximation based on cladistic data (i.e. the discrete morphological characters used to build a phylogenetic tree).
Mathematically, a cladisto-space is an $n$ dimensional object that summarizes the cladistic distances between the taxa present in a cladistic matrix (see details below).
Note that because of its inherent combinatory properties, a cladisto-space is a finite theoretical object limited by the product of the number of character states. Thus a cladisto-space will be overloaded if the number of taxa is higher than the product of the number of character states, although this is not an issue in our study (our cladisto-spaces have maximal capacities of 1.9$\times$$10^{181}$ taxa; \citealp{Slater2012MEE}, and 4.5$\times$$10^{159}$ taxa; \citealp{beckancient2014}). % TG: or 1.9$\times$1.8googol and 4.9$\times$1.6googol. Or is that just useless (basically that is to make the point for one of Gavin's comment: "how do you know you haven't too much taxa to fit in the morphospace?")
% NC: what an odd comment. This may be un-necessary but I'd leave it for now as it's quite cool! It'd be something to cut for PRSB

%Estimating the cladisto-space requires two computationally intuitive % TG: or remove that adjective?
 %steps described in detail below: (1) calculating the distance matrix and (2) the ordination of the distance matrix. % TG: note however that Graeme seems to say the ordination part is useless in cladistic data case. I'm not sure if he's right and I didn't did any testing but his argument make sense: ordination is to summarize the data which is useful with continuous data but less with discrete characters. However statistically I'm not sure how the disparity calculation will work since it is traditionally done on the eigen vectors (PC scores) which is just a "scaling" of the values using the covariance matrix. Therefore, if I want to use just the distance matrix, I'm afraid I'll have to find a complex way to add the covariance to the variance without using the eigen values (that make life easier in this case). Also that'll lead to problems with negative covariance (see Cailliez correction below).
 
 % NC: Yeah I agree with Graeme that ordination on discrete characters is a bit nonsensical. What do people normally use in these caases? Or is this an issue because you're using hypervolumes? If it's the latter I guess you need to chat to Andrew because if it's not a standard method in cladisto-spaces you're going to have problems.

 % NC: OK in response to your email, if it's standard then that's grand.

%\subsubsection{Distance matrix} % I don't think this needs the subheadings
% NC: I like version 2, keeps it simpler.
% NC: Do you have one per study or a joint one? Need to be clearer. I've tried to clarify.
To estimate the cladisto-spaces for each of our datasets we first constructed pairwise distance matrices of length $k$, where $k$ is the total number of taxa in the dataset. 
For each dataset separately, we calculated the $k$$\times$$k$ distances using the Gower distance \citep{Gower71}, i.e. the Euclidean distance between two taxa divided by the number of comparable
% NC: comparable or shared?
characters. 
This allows us to correct for distances between two taxa that share many characters and will be stochastically closer % NC: What does that mean exactly???
to each other than to taxa with fewer characters in common.
For cladistic matrices, using this corrected distance is preferable to the raw Euclidean distance because of its ability to deal with discrete or/and ordinated characters as well as with missing data \citep{anderson2012using}.
However, the Gower distance cannot calculate distances when taxa have no overlapping data.
Therefore, we used the \texttt{TrimMorphDistMatrix} function from the \texttt{Claddis} R package \citep{Claddis} to remove pairs of taxa with no cladistic characters in common.
This led to our removing @/@ taxa from \cite{Slater2012MEE} and @/@ from \cite{beckancient2014}.
%We also calculated the Generalised Euclidean Distance (GED) \citep{Wills2001} which replaces inapplicable distances by the mean overall distance and the MORD distance (@Lloyd in prep.@) which scales the euclidean distance similarly to the Gower distance but by dividing the euclidean distance by the maximum overall distance rather than the number of shared characters. The results of theses calculations are available in the supplementaries % TG: or maybe not. Makes the supplementary super long and it's not something new (to use Gower).
% NC: Yeah leave this out

%\subsubsection{Ordination}
After constructing our distance matrices we transformed them using classical multidimensional scaling \citep[MDS;][]{torgerson1965multidimensional,GOWER01121966,cailliez1983analytical}.
This method (referred to as MDS; e.g. \citealt{DonohueDim}; PCO; e.g. \citealt{Brusatte2015}; or PCoA; e.g. \citealt{paradisape:2004}) is an eigen decomposition of the distance matrix.
Because we used Gower distances instead of raw Euclidean distances, negative eigenvalues can be calculated.
To avoid this, we first transformed the distance matrices by applying the Cailliez correction \citep{cailliez1983analytical} which multiplies the values in a distance matrix (apart from the diagonal) by a constant $c^*$ (as used in \citealt{toljagictriassic-jurassic2013}), 
% NC: and where does c* come from?
We were then able to extract $n$ eigenvectors for each matrix (representing the $n$ dimensions of the cladisto-space) where $n$ is equal to $k-1$, i.e. the number of taxa in the matrix ($k$) minus the last eigenvector which is always null after applying the Cailliez correction.
Contrary to previous studies \citep{brusatte50,cisneros2010,prentice2011,anderson2012using,Hughes20082013,bentonmodels2014}, we use all $n$ dimensions of our cladisto-spaces and not a sub-sample representing the majority of the variance 
% NC: variance in what?
(e.g. 90\% of the variance; \citealt{Brusatte12092008,toljagictriassic-jurassic2013}).
Note that our cladisto-spaces represent an ordination of all the possible mammalian morphologies coded in each study, not the cladistic disparity at any point in time. % NC: I don't get what you mean here.
Therefore, at any point in time, mammalian disparity will be $\leq$ than the total disparity in the cladisto-space. % NC: Need to clarify what you mean here. We probably need to discuss.

\subsection{Estimating disparity}
Disparity can be estimated in many different ways \citep[e.g.][]{Ciampaglio2004,thorneresetting2011,hopkinsdecoupling2013,huang2015origins}, however most studies estimate disparity using four metrics: the sum and products of ranges and variances, each of which gives a slightly different estimate of how the data fits within the cladisto-space \citep{Foote01071994,Wills1994,brusatte50,Brusatte12092008,cisneros2010,thorneresetting2011,prentice2011,brusattedinosaur2012,toljagictriassic-jurassic2013,ruta2013,bentonmodels2014,bensonfaunal2014}.
The sum and products of ranges and variances are based on the ranges and variances of the eigenvectors calculated from a distance matrix.
However, these metrics do not take into account the covariance among eigenvectors.
This is only valid statistically if the eigenvectors are independent.
In multidimensional scaling, all $n$ eigenvectors are calculated from the same distance matrix and are therefore not independent, thus covariances among eigenvectors should be included when estimating disparity.
%Also one practical reason to not use the products of ranges or variances, when using the full cladisto-space (i.e. without excluding the last dimensions), is that the disparity values will tend towards 0 since the variance or ranges on the last axis are usually really close to 0. % TG: or maybe that's useless. We're already making the point that's it statistically incorrect. No need to add that it also doesn't work in practice, no?
% NC: Yes not sure how easy it is to make this point clearly and succinctly so leave it out.

% NC: Note that this section is KEY! You are going to get grumpy paleo types insisting you use the old methods. Need to really clearly explain WHY you chose not to.

Unfortunately, including the covariance terms in these measurements
% NC: Which measurements? Disparity? eigenvectors
is not straightforward and comes with properties specific to the number of dimensions involved \citep{Jackson,DonohueDim}. % NC: What do you mean it "comes with properties". Be clear.
%One potential solution is to measure the hyper-dimensional volume \citep{Wills1994,DonohueDim} but this volume based on the eigenvalue 
% NC: eigenvalue or eigenvalues? Of the distance matrix?
%cannot be calculated through time (unless one recalculates the cladisto-space at each time point).
% NC: Why mention all this if you're not going to use hypervolumes?
Therefore, we chose to use a more intuitive metric for measuring the dispersion of the data in the cladisto-space: the distance from centroid (\citealt{finlay2015morphological}; similar but not equivalent to \citealt{kornextinction2013,huang2015origins}):
%\begin{equation}
%Disparity=\frac{\sum{\sqrt{(kn_{i}-Centroid_{n})^2}}}{k}
%\end{equation}
%Where $k_{n}$ is any value of eigen vector of the $n^{th}$ dimension of the cladisto-space and $Centroid_{n}$ is the average value of the same $n^{th}$ eigen vector and $k$ is the size of the distance matrix (the total number of taxa).

  \begin{equation}
    Disparity=\frac{\sum{\sqrt{(Edge_{n}-Centroid_{n})^2}}}{Number\ of\ edges}
  \end{equation}

Where $Edge_{n}$ is any phylogenetic edge value % NC: What's a phylogenetic edge value? OK from reading later it appears this is a confusion. Change to something else. Maybe go back to the letters?
present in the subsection of data in the $n^{th}$ dimension of the cladisto-space % NC: What's that?
and $Centroid_{n}$ is the average value of the $n^{th}$ dimension. % NC: This needs to be explained far more clearly.
We also estimated the sum and products of ranges and variances to compare our results with previous studies. % link to supp.

\subsection{Estimating disparity through time} % NC: I've tried to make this section more concise.
Changes in disparity through time are generally investigated by estimating the disparity of taxa that occupy the cladisto-space during specific time intervals \citep[e.g][]{cisneros2010,prentice2011,Hughes20082013,hopkinsdecoupling2013,bentonmodels2014,bensonfaunal2014}.
These time intervals are usually defined based on biostratigraphy \citep[e.g.][]{cisneros2010,prentice2011,Hughes20082013,bentonmodels2014} but can also be arbitrarily chosen time periods of equal duration \citep{hopkinsdecoupling2013,bensonfaunal2014}.
However, this approach suffers from two main biases. 
First, if biostratigraphy is used to determine the time intervals, disparity may be distorted towards higher differences between time intervals because biostratigraphal periods are geologically defined based on differences in the morphology of fossils found in the different strata.
Second, the approach assumes that all characters evolve following a punctuated equilibrium model, because disparity is only estimated once for each interval resulting in all changes in disparity occurring between intervals, rather than also allowing for gradual changes within intervals \citep{Hunt21042015}.

To address these issues, we use a ``time-slicing'' approach that considers subsets of taxa in the cladisto-space at specific equidistant points in time, as opposed to considering subsets of taxa between two time points %(see Fig.~\ref{fig_slicing}) NC: I don't think this figure helps as it is at present
.
This results in even-sampling of the cladisto-space across time and at a finer grain than using time intervals, and permits us to define the underlying model of character evolution (as punctuated or gradual).  
In practice, time-slicing considers the disparity of any element present in the phylogeny (branches, nodes and tips) at any point in time %(see Fig.~\ref{fig_slicing}) NC: I don't think this figure helps as it is at present
.
When the phylogenetic elements are nodes or tips, the eigenvector scores for the nodes (estimated using ancestral state reconstruction as described above) or tips are directly used for estimating disparity.
When the phylogenetic elements are branches we inferred the eigenvector score for the branch using one of two evolutionary models:
\begin{enumerate}
    \item{\textbf{Punctuated evolution.}} 
    This model selects the eigenvector score from either the ancestral node or the descendant node/tip of the branch. 
    Similarly to the time interval approach, this reflects a model of punctuated evolution where changes in disparity occur either at the start or at the end of a branch over a relatively short time period \citep{Gould1977}.
    We applied this model in three ways: 
    \begin{enumerate}[(i)]
      \item selecting the eigenvector score of the ancestral node of the branch
      \item selecting the eigenvector score of the descendant node/tip of the branch
      \item randomly selecting either the eigenvector score of the ancestral node or the descendant node/tip of the branch
    \end{enumerate}
    Method (i) assumes that changes always occurs early on the branch (accelerated transition, ACCTRAN) and (ii) assumes that changes always occur later (delayed transition, DELTRAN).
    We prefer not to make either assumption so we report the results from (iii), although the ACCTRAN and DELTRAN results are available in the Supplementary Information. % link
    \item{\textbf{Gradual evolution.}}
    This model also selects the eigenvector score from either the ancestral node or the descendant node/tip of the branch, but the choice depends on the distance between the sampling time point and the end of the branch.
    If the sampling time point falls in the first half of the branch length the eigenvector score is taken from the ancestral node, conversely, if the sampling time point falls in the second half of the branch length the eigenvector score is taken from the descendant node/tip.
    This reflects a model of gradual evolution where changes in disparity are gradual and cumulative along the branch.
\end{enumerate}

% TG: not sure if this figure is useful and pretty sure that if it is, it should be improved!
%\begin{figure}[!htbp]
%\centering
%    \includegraphics[keepaspectratio=true]{Figures/Slicing.pdf}
%\caption{Differences in slicing method and intervals. Solid lines represent the First and Last Appearance Datum span. Interval 1 contains the following elements within the global character-space: taxa t2 and t8 and nodes n1, n2, n3, n4 and n5. Interval 2 contains the following elements within the global character-space: taxa t1, t2, t3, t4, t5, t6 and t7 and nodes n6 and n7. Slice 1 contains the following elements within the global character-space under the proximity model (constant evolution assumption): nodes n2 and n4. Slice 2 contains the following elements within the global character-space under the proximity model (constant evolution assumption): taxa t4 and t7.}
%\label{fig_slicing}
%\end{figure}
% NC: I don't think this figure helps as it is at present. For the thesis it might be nice to have a super simple one just with dots or crosses on the branches and tips where your slice overlaps.

%What did we do (the main figure)
We applied our time-slicing approach to the two cladisto-spaces calculated from \cite{Slater2012MEE} and \cite{beckancient2014}, time-slicing the phylogeny every five million years from 170 Mya to the present resulting in 35 sub-samples of the cladisto-space.
For each sub-sample, we estimated its disparity assuming punctuated (ACCTRAN, DELTRAN and random) and gradual evolution as described above.
To reduce the influence of outliers on our disparity estimates % NC: This is what this does right? not really uncertainty as in your original draft.
, we bootstrapped each disparity measurement 1000 times using...  % NC: What kind of bootstrapping?
We then calculated the median disparity value for each sub-sample along with the 50\% and the 95\% confidence intervals.

We also recorded 
%the taxonomic richness % NC: I think this needs a different name. Taxonomic richness is already a thing. Maybe just number of phylogenetic elements? It's easily solved here be shortening but may be a problem later?
the number of phylogenetic elements (branches, nodes and tips) in each sub-sample.
% NC: Why log? Maybe leave that out for now and put it in where necessary?
Disparity may be higher in sub-samples with more phylogenetic elements simply because there are more taxa represented. To test whether our analyses were biased in this way, we ran a rarefaction analysis where we randomly re-sampled each sub-sample so that each had the same number of phylogenetic elements. We then estimated disparity and bootstrapped as described above. We repeated this XX times. % NC: Not sure if this is what you did, but standard rarefaction involves repeated re-samplings. Or else you can get order effects etc.
We report the results of in the Supplementary Materials. % NC: Though you probably need to state in the results that this didn't cause the results - same goes for all your supplemental analyses.

To compare our results to previous studies we also repeated our analyses using two time interval approaches; one based on biostratigraphy \citep[e.g.][]{cisneros2010,prentice2011,Hughes20082013,bentonmodels2014} using each geological stage from the Middle Jurassic to the present, and one using equal time periods of 20 million years \citep{hopkinsdecoupling2013,bensonfaunal2014}.
We report the results of these analyses in the Supplementary Materials. % link

% NC: I've combined the two as the long version wasn't all that much longer than the short version so it's a bit pointless having it in supplemental. Of course the results go there. I've also moved it around a bit to make it flow better.

% NC: I don't really understand what you mean below, but hopefully what I've written above expresses what you actually did?

% Each subdivision was therefore re-sampled to contain the minimal number of taxa in the smallest subdivision.
% Because this led in certain cases to a absolute minimum of phylogenetic elements to calculate disparity we also re-sampled the subdivisions to contain at maximum the modal number of phylogenetic elements between each sub-sample of the cladisto-space.
% In practice, for both cases, we bootstrapped the disparity measurements by randomly removing $x$ elements per subdivision, where $x$ is either (1) the number of edges sampled in each time slice minus the minimum of edges in each time slice or (2) the number of edges to remove so that the number of elements is not higher than the modal number of edges.
% Both results are available in the supplementary materials.

%Additionally, in order to make our results comparable with previous studies, we also calculated the sum and products of ranges and variances \citep{Foote01071994,Wills1994} on the sub-samples of the cladisto-space. % NC: You've already mentioned this above - leave it out.

\subsection{Testing the effect of the K-Pg boundary on mammalian disparity} % NC: or similar, something more descriptive than "null model testing" anyway.
Once we had estimated disparity through time for each of our cladisto-spaces we used null models to test whether our observed changes in mammalian disparity at the K-Pg boundary were significantly greater than expected by chance.
We generated random character matrices under two null models as follows.
  \begin{enumerate}
    \item Random. We constructed a purely stochastic character matrix where each cell value was sampled from a discrete uniform distribution with a number of states equal to the number of states observed % NC: for that cell? overall? average?
    in the original data. We repeated this 100 times to get 100 random character matrices.
    \item Brownian. % NC: You refer to this as brownian earlier but it's not really is it... Can't work out what the word would be but we can look it up!
    We simulated each character for each node or tip using the \texttt{sim.character} function from the \texttt{diversitree} R package \citep{fitzjohndiversitree2012} under the $Mk_n$ model \citep{lewisa2001} with an equal transition rate between the number of characters states observed in the original matrix and a unique rate ($\mu$) sampled from a uniform distribution (0 $<$ $\mu$ $\leq$ 0.5). % NC: Maybe be a bit clearer about what the parameters are here
    We repeated this 100 times to get 100 random null character matrices. % NC: I'm guessing you repeated this?
  \end{enumerate}
We reran our disparity analyses (minus the ancestral states reconstructions as this would be nonsensical with simulated data) for each of our randomly generated character matrices following the same steps as described above. % NC: I can't decide if this is clear but I think it is clear that you did it the exact same way and for both phylogenies.

We then measured the amount of overlap between both null models
% NC: What exactly are you measuring? disparity through time? disparity at kpg?
and the observed data using the Bhattacharyya Coefficient \citep{Bhattacharyya} which measures the probability of overlap between two distributions. 
This is similar to a two-sided \textit{t-test} but captures differences across the whole distributions not just the means. % NC: I don't think we can cite ourselves here!
Bhattacharyya Coefficients \textgreater 0.95 suggest that the observed disparity values are no different than expected under the null model, whereas Bhattacharyya Coefficients \textless 0.05 suggest that there is a significant difference between observed disparity values and the null.
% OR actually maybe just go for anova... The problem is again that for our null data we have the full bootstrapped distribution (1000000 data points) to compare to the observed bootstrapped distribution (1000 points). Frequentist test will be a bit shitty (anyway because of the huge difference between the two distributions (observed and null) the results will be significant even if I only sample 10 data points in each distributions...). Maybe just add both? t-test + Bhattacharrya?

% NC: Agreed, ANOVA and t tests are going to give a sig difference, but could include them anyway. But worth thinking about WHAT is the test you are doing here? You're actually with bhattacharrya looking at the whole distribution. So you're asking, is there a difference in how diversity changes through time in null and non null. Really the question is "is there a difference at KPg". so perhaps you need to pool data or something? We can discuss.
%---------------------------------------------
%
%       RESULTS
%
%---------------------------------------------

\section{Results}

%---------------------------------------------
%
%       DISCUSSION
%
%---------------------------------------------

\section{Discussion}

%Disparity improvements
%Previous studies have calculated disparity on a subset of PCO axes \citep[e.g.][]{Brusatte12092008} but in this study we calculated it on all the available axis (i.e. the full n dimensional cladisto-space) to avoid excluding outliers.

%Biases:
%-internal versus terminal branches
%-ancestral states reconstruction (solved by being conservative?)
%-poor sampling of living taxa (Guillerme & Cooper)

%From Wilson 2013
%My results reveal several key findings: (1) latest Cretaceous mammals, particularly metatherians and multituberculates, had a greater ecomorphological diversity than is generally appreciated, occupying regions of the morphospace that are interpreted as strict carnivory, plant-dominated omnivory, and herbivory; (2) the decline in dental-shape disparity and body-size disparity across the K/Pg boundary shows a pattern of constructive extinction selectivity against larger-bodied dietary specialists, particularly strict carnivores and taxa with plant-based diets, that suggests the kill mechanism was related to depressed primary productivity rather than a globally instantaneous event; (3) the ecomorphological recovery in the earliest Paleocene was fueled by immigrants, namely three multituberculate families (taeniolabidids, microcosmodontids, eucosmodontids) and to a lesser extent archaic ungulates; and (4) despite immediate increases in the taxonomic richness of eutherians, their much-celebrated post-K/Pg ecomorphological expansion had a slower start than is generally perceived and most likely only began 400,000 to 1 million years after the extinction event.


%Therefore, the cladisto-space is likely to be more influenced by phylogeny than the morphospace \citep{Foote29111996,Wagner01011997}. However, discrete cladistic characters are still the best source for quantifying overall morphology for large and diverse groups \citep{Brusatte12092008}. % NC: Make this justification better. why does the phylogney effect matter. Should this be mentioned here at all? TG: not specially, might be something a reviewer might point out. I remember you pointing it out ;). However, STD is become fairly common these days so maybe people have accepted the idea.

% Placental vs eutherian? Cartmill blogpost

%---------------------------------------------
%
%       CONCLUSION
%
%---------------------------------------------

\section{Conclusion}
%Not much stuff is happening there but we bring improvement to methods.


%---------------------------------------------

\section{Data availability and reproducibility}

\section{Acknowledgments}
Graeme Lloyd, Andrew Jackson, Gavin Thomas, Sive Finlay.
%Simulations used the Lonsdale cluster maintained by the Trinity Centre for High Performance Computing and funded through grants from Science Foundation Ireland. %TG: I think they won't in the end

\section{Funding} % NC: Usually this is part of acknowledgments.
This work was funded by a European Commission CORDIS Seventh Framework Programme (FP7) Marie Curie CIG grant (proposal number: 321696).

 %   \citept{key} ==>>                Jones et al. (1990)
 %   \citept*{key} ==>>               Jones, Baker, and Smith (1990)
 %   \citep{key} ==>>                (Jones et al., 1990)
 %   \citepp*{key} ==>>               (Jones, Baker, and Smith, 1990)
 %   \citepp[chap. 2]{key} ==>>       (Jones et al., 1990, chap. 2)
 %   \citep[e.g.][]{key} ==>>        (e.g. Jones et al., 1990)
 %   \citepp[e.g.][p. 32]{key} ==>>   (e.g. Jones et al., p. 32)
 %   \citepauthor{key} ==>>           Jones et al.
 %   \citepauthor*{key} ==>>          Jones, Baker, and Smith
 %   \citepyear{key} ==>>             1990

\bibliographystyle{sysbio}
\bibliography{References}

\section{supplementaries}

\subsection{Ancestral states estimation}
We used both the \texttt{ace} function from the R package ape v. 3.2 \citep{paradisape:2004} and the 
\texttt{rerootingMethod} function from the R package phytools 0.4-45 \citep{phytools}. Both method perform a maximum likelihood estimation of the ancestral values and the variance of a Brownian motion process based on the re-rooting method of \citep{Yang01121995}. The two methods differ slightly in the calculation of the normalized conditional likelihoods but mainly on the way to treat missing data. We optimised the \texttt{ace} function for fast estimation by treating missing data in the matrix as an extra character (e.g. if a character has two observed tips states 0 and 1 and a third tip has missing data (NA), the ancestor of these three tips can be estimated between the three following states: 0, 1 and NA). For the \texttt{rerootingMethod}, we followed \citep{Claddis} method and treated the missing in the tips as any possible observed state (e.g. if a character has two observed tips states 0 and 1 and a third tip has missing data (NA), the third tip will be considered as multi-state (0\&1) and the ancestor of these three tips can be estimated between the two following states: 0 and 1). Both methods perform similarly but the implementation of the \texttt{ace} function has a slightly lower accuracy  but is three times faster than the one for the \texttt{rerootingMethod} function (see supplementaries).
% NC: Some of this probably belongs in methods

\subsubsection{Time intervals}
We then divide our observed cladisto-spaces into sub cladisto-spaces representing the different stages of the character-space filling. For example, if at various points in time.
%The intervals should be a compromise between the resolution and the sample size and must be "sufficiently coards that nearly all generic first and last occurenaces can be unambiguously assigned" \citep{Foote01071994}.
Time intervals from 170Ma (Earliest Cenomanian, Late Cretaceous) to the present.
We count all the nodes/tips present in a given time interval.
Classic but artificially grouping data. The minimal bin size should contain at least two nodes/tips and sometime that involves having time intervals spanning accross tens of millions of years. Such long duration time intervals have no real biological meaning since it is unlikely that all of the nodes/tips present in the time interval did ever coexisted and had ever biological interactions together.

\subsection{Diversity}
-Diversity in living mammals
-Diversity per interval

\subsection{Disparity}
-Centroid is less correlated with diversity
-Other metrics

\subsection{Not to be in the paper, neither in the supplementaries (methods table)}

\begin{table}[ht]
\caption{Comparison of Cladisto-space studies methods}
\centering
\begin{tabular}{cccccccc}
  \hline
    Date & Author      & Distance  & axis & Binning    & Disparity   & Difference & cite \\ %
  \hline
         & this study  & Gower     & PCO        & Time slice & centroid    & NPMANOVA?  & \\
    2014 & Benson      &           &            & Equal bins & Wills 1994* & NPMANOVA   & \citep{bensonfaunal2014} \\
    2014 & Brusatte    & Euclidean & PCO        &            &             &            & \citep{brusattegradual2014} \\
    2014 & Benton      & Euclidean & PCO        & Biostrat   & Wills 1994* & NPMANOVA   & \citep{bentonmodels2014} \\
    2013 & Hopkins     &           &            & Equal bins & Wills 1994* &            & \citep{hopkinsdecoupling2013} \\             
    2013 & Ruta        & GED       & 10 first   & Biostrat   & Wills 1994* & NPMANOVA   & \citep{ruta2013} \\
    2013 & Hughes      & Euclidean &            & Biostrat   & Sum of var  &            & \citep{Hughes20082013} \\
    2013 & Toljagic    & Euclidean & 90\% var   & Biostrat   & Wills 1994* & NPMANOVA   & \citep{toljagictriassic-jurassic2013} \\
    2012 & Brusatte    & Euclidean & 90\% var   & Biostrat   & Wills 1994* & CI overlap & \citep{brusattedinosaur2012} \\
    2012 & Anderson    & Gower     & PCO        &            &             &            & \citep{anderson2012using} \\
    2010 & Prentice    & Euclidean & PCO        & Biostrat   & Wills 1994* & NPMANOVA   & \citep{prentice2011} \\
    2011 & Thorne      & Euclidean &            & Biostrat   &             & NPMANOVA   & \citep{thorneresetting2011} \\
    2010 & Cisneros    & Euclidean & PCO        & Biostrat   & Wills 1994* & NPMANOVA   & \citep{cisneros2010} \\
    2008 & Brusatte    & Euclidean & 90\% var   & Biostrat   & Wills 1994* & NPMANOVA   & \citep{brusatte50} \\
    2008 & Brusatte    & Euclidean & 90\% var   & Biostrat   & Wills 1994* & NPMANOVA   & \citep{Brusatte12092008} \\
    2005 & Wesley-Hunt &           & PCO        &            & Foote 1992  & t-test     & \citep{Wesley-Hunt2005} \\
  \hline
\end{tabular}
\end{table}
* The 4 sum and product of range and variance




\end{document}
